config/default/akamai-edgerc-secret.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: akamai-edgerc-secret
+stringData:
+  AKAMAI_HOST: ${AKAMAI_HOST}
```

Comment:
I think all of these have to have defaults so it is not a required field with clusterctl initialization

Reply:
I've added a ```""``` default now